### PR TITLE
Add initial draft of git background

### PR DESCRIPTION
**Pull request by NLesniak**
Created background and setup git section
TO DO:
Update images
Edit to include more examples and better explanations where appropriate

### CODE REVIEW

File: _episodes/03-intro-git-github.md [hunk 1]
```diff
@@ -1,15 +1,248 @@
...
+| Notepad++ (Win, 32-bit install)    | `$ git config --global core.editor "'c:/program files (x86)/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
+| Notepad++ (Win, 64-bit install)    | `$ git config --global core.editor "'c:/program files/Notepad++/notepad++.exe' -multiInst -notabbar -nosession -noPlugin"`|
+| Kate (Linux)       | `$ git config --global core.editor "kate"`       |
+| Gedit (Linux)      | `$ git config --global core.editor "gedit --wait --new-window"`   |
+| Scratch (Linux)       | `$ git config --global core.editor "scratch-text-editor"`  |
+| Emacs              | `$ git config --global core.editor "emacs"`   |
+| Vim                | `$ git config --global core.editor "vim"`   |
+| VS Code                | `$ git config --global core.editor "code --wait"`   |
+
+It is possible to reconfigure the text editor for Git whenever you want to change it.
```
Comment (kelly-sovacool) [inline _episodes/03-intro-git-github.md:188]: Should we add a statement recommending `nano` if they haven't installed or used the other options before?

**Review by kelly-sovacool — COMMENTED** [review]
In my branch (PR #17)  I copied all of the figures from git-novice to `fig/git/`, do you want to use that path? We can pick a different path name btw if you have another idea. (cc @NLesniak @diaorch)

**Comment by NLesniak** [comment]
@kelly-sovacool  @diaorch  I think `fig/git/` makes sense, that way we can maintain organization across the different parts, I will fix my paths and remove the figures

@kelly-sovacool  as for Vim, i wasn't sure if I should remove the Vim part and add nano

**Review by kelly-sovacool — APPROVED** [review]
Looks great!

**Pull request merged**